dts/arm/ene/kb1200.dtsi

```diff
@@ -335,6 +335,40 @@
 			#size-cells = <0>;
 			status = "disabled";
 		};
+
+		bbram: bbram@40041000 {
+			compatible = "ene,kb1200-bbram";
+			reg = <0x40041000 0x84>;
+			bbram-size = <124>;
```

Comment:
what is the difference between `bbram-size` and the size cell of the `reg` property here? I see the value is slightly different but similar